Source/GR32.pas

```diff
@@ -6361,7 +6361,11 @@ procedure TCustomBitmap32.LoadFromFile(const FileName: string);
 {$endif LOADFROMSTREAM}
 begin
 {$ifndef FPC}

```

Comment:
Please change this conditional block to:
```
{$if (not defined(FPC)) and (CompilerVersion >= 31.0)} // TBufferedFileStream was introduced in Delphi 10.1
  FileStream := TBufferedFileStream.Create(FileName, fmOpenRead or fmShareDenyWrite);
{$else}
  FileStream := TFileStream.Create(FileName, fmOpenRead or fmShareDenyWrite);
{$ifend}
```

Reply:
that's a much simpler solution--I forget that "and" can be use in compiler directives